libs/spirit-islander/feature-config/src/lib/config.component.ts

```diff
@@ -0,0 +1,35 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { first, map } from 'rxjs';
+
+import { AppStateService } from '@atocha/spirit-islander/data-access';
+import {
+  ConfigDetails,
+  ConfigFormComponent,
+} from './config-form/config-form.component';
+
+@Component({
+  standalone: true,
+  selector: 'app-config',
+  imports: [CommonModule, ConfigFormComponent],
+  template: `
+    <app-config-form
+      *ngIf="config$ | async as config"
+      [config]="config"
+      (generate)="onGenerate($event)"
+    ></app-config-form>
+  `,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfigComponent {
+  config$ = this._appStateService.state$.pipe(
+    first(),
+    map(({ config }) => config)
+  );
+
+  constructor(private _appStateService: AppStateService) {}
+
+  onGenerate({ config }: ConfigDetails): void {
```

Comment:
If `validCombos` no longer needs to be passed around, the form component could just emit the `config` object

Reply:
Mentioned [here](https://github.com/johnnycopes/atocha/issues/280#issuecomment-1490279937)